src/cli/README.md

```diff
@@ -0,0 +1,36 @@
+# Command Line Interface
```

Comment:
as a generate feedback, I think the CLI interface would be more useful if it was using a version of the platform stub that send real udp/tcp/http messages, that would allow developers to test interactively against their own device w/o deploying to a Google Home device.

Reply:
I think I'd agree to add in radios unless we use some sort of argument not to use radios (like `--simulate`) to suggest that the messages aren't real.

Reply:
Will look into this

Reply:
See #47